svelte/components/Nft/NftMintButton.svelte

```diff
@@ -109,6 +126,8 @@
           <em>{nftMintTexts[minting]}</em>
         </div>
       {/if}
+    {:else if nid}
+      <button on:click={nftLink} class="btn btn-small btn-sell" title="View in Explorer">NFT LINK</button>
```

Comment:
nice button !

"VIEW NFT" instead of “NFT LINK"

Reply:
should also display "MINTING 1/4" "MINTING 2/4" "MINTING 3/4" "MINTING 4/4" 

instead of "MINTING 1..."

Reply:
add steps count according to "nftMintTexts" array's length